app/src/main/java/org/glucosio/android/adapter/A1cEstimateAdapter.java

```diff
@@ -43,12 +40,12 @@ public View getView(int position, View convertView, ViewGroup parent) {
         A1cEstimate p = getItem(position);
 
         if (p != null) {
-            TextView value = (TextView) v.findViewById(R.id.dialog_a1c_item_value);
-            TextView month = (TextView) v.findViewById(R.id.dialog_a1c_item_month);
-            TextView glucoseAverage = (TextView) v.findViewById(R.id.dialog_a1c_item_glucose_value);
+            TextView value = v.findViewById(R.id.dialog_a1c_item_value);
+            TextView month = v.findViewById(R.id.dialog_a1c_item_month);
+            TextView glucoseAverage = v.findViewById(R.id.dialog_a1c_item_glucose_value);
 
             if (value != null) {
-                if ("percentage".equals(db.getUser(1).getPreferred_unit_a1c())) {
+                if ("percentage".equals(databaseHandler.getUser(1).getPreferred_unit_a1c())) {
```

Comment:
The app supports multiple users? If not why need the parameter? Also, could create a default method without parameter where internally could call `getUser(1)` something called as `getDefaultUser()`.

Reply:
That I don't know. I'm not sure about the original idea. Currently, it is only one user